x-pack/platform/plugins/shared/observability_ai_assistant/server/service/client/index.ts

```diff
@@ -210,33 +209,9 @@ export class ObservabilityAIAssistantClient {
           });
         }
 
-        const userInstructions$ = from(this.getKnowledgeBaseUserInstructions()).pipe(shareReplay());
-
         const registeredAdhocInstructions = functionClient.getAdhocInstructions();
```

Comment:
I think we can get rid of `getAdhocInstructions ` and `registerAdhocInstruction` and just use `getInstructions` and `registerInstruction` instead.